src/Faker/Provider/Internet.php

```diff
@@ -204,21 +206,23 @@ public function ipv6()
      */
     public static function localIpv4()
     {
-        if (static::numberBetween(0, 1) === 0) {
-            // 10.x.x.x range
-            return long2ip(static::numberBetween(ip2long("10.0.0.0"), ip2long("10.255.255.255")));
-        }
+        $ipBlocks = [
```

Comment:
Can you please add a reference to the corresponding RFC document? (https://tools.ietf.org/html/rfc1918#section-3)